Wikipedia/Code/WMFProxyServer.m

```diff
@@ -95,8 +95,8 @@ - (GCDWebServerAsyncProcessBlock)defaultHandler {
                        return;
                    }
 
-                   if (!([originalSrc hasPrefix:@"http"] || [originalSrc hasPrefix:@"https"])) {
-                       originalSrc = [@"http:" stringByAppendingString:originalSrc];
+                   if ([originalSrc hasPrefix:@"//"]) {
+                       originalSrc = [@"https:" stringByAppendingString:originalSrc];
```

Comment:
@montehurd I made this change because it looked like on en.wiki it would just make those requests and be redirected to HTTPS. Also, security wise, this would prevent someone from sniffing which articles are being viewed based on image URLs. Might this be an issue on other wikis/should we build in a param that passes whether or not the wiki supports HTTPS?